lang/ucblogo/Portfile

```
@@ -1,38 +1,26 @@
-PortSystem 1.0
-
-name             ucblogo
-version          6.2.2
-revision         0
-categories       lang
-maintainers      nomaintainer
-platforms        darwin
-description      Berkeley Logo
-long_description \
-    Berkeley Logo is a free Logo interpreter.
-homepage         https://people.eecs.berkeley.edu/~bh/logo.html
-
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                ucblogo
+github.setup        jrincayc ${name}-code 6.2.2
+revision            1
+
+github.tarball_from archive
+distname            ${github.project}-version_${github.version}
+checksums           rmd160  2bb21204cd82fe5fdf3d7d9eb05df465d1ad5f6c \
+                    sha256  7269120e395c6745989e5c12b7023e554b8826926c863da2bf522436f5f5921a \
+                    size    823802
+
+categories          lang
+maintainers         nomaintainer
+description         Berkeley Logo
+long_description    Berkeley Logo is a free Logo interpreter.
+homepage            https://people.eecs.berkeley.edu/~bh/logo.html
 # Note that license has changed to GPL-3 in 6.1
-license          GPL-3
-
-master_sites     ftp://ftp.cs.berkeley.edu/pub/ucblogo/
-distfiles        ${name}.tar.gz
-checksums        rmd160  7cd9ec8eeff00fb3440f5036d57a9b29383ab310 \
-                 sha256  c659e989e9284efc0f904ebd7de24b559c2d7434a98bb9c92c23e6c11b49cef3 \
-                 size    6694879
-
-fetch.use_epsv   no
-
-patchfiles       patch-makefile.in.diff \
-                 patch-logo_h.diff \
-                 patch-wrksp_c.diff
-
-post-patch {
-	file delete -force "${worksrcpath}/csls/CVS"
-}
+license             GPL-3
 
-configure.args   --cache-file=/dev/null --without-x
+use_autoreconf      yes
+autoreconf.args     --install
 
-build.target
+configure.args      --cache-file=/dev/null --without-x --disable-docs
```

This project uses ncurses, and if MacPorts ncurses is installed, it will use it opportunistically. Better to add the dependency.
```suggestion
depends_lib         port:ncurses

configure.args      --cache-file=/dev/null --without-x --disable-docs
```

---

Hi @ryandesign, 

I have added this and updated my commit.

Thanks,

-ranga